tsconfig.json

```diff
@@ -7,7 +7,6 @@
         "sourceMap": true,
         "strict": true
     },
-    "files": ["src/server/server.ts"],
-    "include": [],
+    "include": ["src"],
```

Comment:
this fixes the "not found" errors, e.g.
`Parsing error: /Users/tom_forbes/code/support-dotcom-components/src/shared/types/targeting/shared.ts was not found by the project service. Consider either including it in the tsconfig.json or including it in allowDefaultProject`